lighthouse-core/lib/log.js

```diff
@@ -113,23 +115,23 @@ class Log {
     Log._logToStdErr(`${prefix}:${level || ''}`, [data.method, snippet]);
   }
 
-  static log(title) {
-    Log.events.issueStatus(title, arguments);
-    return Log._logToStdErr(title, Array.from(arguments).slice(1));
+  static log(title, ...args) {
+    Log.events.issueStatus(title, ...args);
```

Comment:
There is one difference with the previous code block.
...args doesn't contain title whereas arguments did.
to fix you should do `issueStatus(title, [title, ...args])`